<launch>
  <arg name="robot_name" default="/"/>
  
  <group ns="$(arg robot_name)">
    <node pkg="multirobot_stage" type="global_odom" name="global_odom" output="screen" respawn="true"/>
  </group>
</launch>
